modules/charon-core/src/main/java/org/wso2/charon3/core/encoder/JSONEncoder.java

```diff
@@ -97,7 +98,8 @@ public String encodeSCIMException(AbstractCharonException exception) {
 
         try {
             //construct error object with details in the exception
-            errorObject.put(ResponseCodeConstants.SCHEMAS, new String[]{exception.getSchemas()});
+            JSONArray arrSchemas = new JSONArray(Arrays.asList(new String[]{exception.getSchemas()}));
```

Comment:
```suggestion
            JSONArray arrSchemas = new JSONArray(Collections.singletonList(exception.getSchemas()));
```
Shall we simplify the array generation?